cxx-squid/src/test/resources/metrics/public_api.h

```diff
@@ -45,6 +45,9 @@ class testClass
 	A(A const&) = delete;
 
 	// ignore defaulted methods
+  /**
+  operator= doc
+  */
```

Comment:
I did some more analysis and the algorithm to detect the "= default" fails because of C++/CLI extension
```
    b.rule(unqualifiedId).is(
      b.firstOf(
        templateId,
        IDENTIFIER,
        operatorFunctionId,
        conversionFunctionId,
        literalOperatorId,
        b.sequence("~", className),
        b.sequence("~", decltypeSpecifier),
        b.sequence("!", className),
        cliGenericId,
        CxxKeyword.DEFAULT
        )
      );
```
the AST looks now different:
![capture](https://cloud.githubusercontent.com/assets/2315215/8735074/65221ab0-2c0e-11e5-8639-017ea9c90c9f.PNG)
Removing this extension of *unqualfied ID* creates several parser errors in C++/CLI code :worried: